samples/cors/ts/server/server.ts

```diff
@@ -0,0 +1,47 @@
+import express from "express";
+import dotenv from "dotenv";
+import cors from "cors";
+import { DefaultAzureCredential } from "@azure/identity";
+import { SecretClient } from "@azure/keyvault-secrets";
+
+// Pull in azure credentials from a .env file
+// For more information on dotenv please refer to https://www.npmjs.com/package/dotenv
+dotenv.config();
+
+const PORT = process.env.PORT || 4000;
+
+if (!process.env.AZURE_KEYVAULT_NAME) {
+  throw new Error("AZURE_KEYVAULT_NAME is missing or empty in .env file");
+}
+const KEYVAULT_URI = `https://${process.env.AZURE_KEYVAULT_NAME}.vault.azure.net/`;
+const client = new SecretClient(KEYVAULT_URI, new DefaultAzureCredential());
+
+const app = express();
+
+// Apply the default CORS policy which would allow all CORS requests.
+// While convenient for development it is overly permissive and we encourage
+// you to review the documentation and apply a CORS policy that makes sense
+// for your application.
+app.use(cors());
```

Comment:
Maybe worth just adding in a default (that won't work) and that they'll need to fill in?

```suggestion
// this came straight from the express cors docs
var corsOptions = {
  origin: 'http://example.com',
  optionsSuccessStatus: 200 // some legacy browsers (IE11, various SmartTVs) choke on 204
}

app.use(cors(corsOptions));
```

Reply:
Thanks! I updated the cors policy, restricting it to the default origin used by parcel